terms/experimentalData/metaboliteType.json

```
@@ -124,39 +144,19 @@
             "source": "http://purl.bioontology.org/ontology/MESH/D013261"
         },
         {
-            "const": "endocannabinoids",
-            "description": "Fatty acid derivatives that have specificity for CANNABINOID RECEPTORS. They are structurally distinct from CANNABINOIDS and were originally discovered as a group of endogenous CANNABINOID RECEPTOR AGONISTS.",
-            "source": "http://purl.bioontology.org/ontology/MESH/D063388"
-        },
-        {
-            "const": "oxylipins",
-            "description": "Eighteen-carbon cyclopentyl polyunsaturated fatty acids derived from ALPHA-LINOLENIC ACID via an oxidative pathway analogous to the EICOSANOIDS in animals.",
-            "source": "http://purl.bioontology.org/ontology/MESH/D054883"
-        },
-        {
-            "const": "eicosanoids",
-            "description": "A class of compounds named after and generally derived from C20 fatty acids (EICOSANOIC ACIDS) that includes PROSTAGLANDINS; LEUKOTRIENES; THROMBOXANES, and HYDROXYEICOSATETRAENOIC ACIDS.",
-            "source": "http://purl.bioontology.org/ontology/MESH/D015777"
-        },
-        {
-            "const": "peptides",
-            "description": "Members of the class of compounds composed of AMINO ACIDS joined together by peptide bonds between adjacent amino acids into linear, branched or cyclical structures.",
-            "source": "http://purl.bioontology.org/ontology/MESH/D010455"
+            "const": "sugars",
+            "description": "Short chain carbohydrate molecules that have hydroxyl groups attached to each carbon atom unit with the exception of one carbon that has a doubly-bond aldehyde or ketone oxygen.",
+            "source": "http://purl.bioontology.org/ontology/MESH/D000073893"
         },
         {
-            "const": "cofactors and vitamins",
-            "description": "Those reactions involved in the synthesis, utilization and/or degradation of cofactors, vitamins and other nutrients.",
-            "source": "http://bioportal.bioontology.org/ontologies/PW/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPW_0000135"
+            "const": "TCA cycle metabolites",
+            "description": "TCA cycle metabolites",
+            "source": "http://www.semanticweb.org/ADMO#TCACycle"
         },
         {
             "const": "xenobiotics",
             "description": "Chemical substances that are foreign to the biological system. They include naturally occurring compounds, drugs, environmental agents, carcinogens, insecticides, etc.",
             "source": "http://purl.bioontology.org/ontology/MESH/D015262"
-        },
-        {
-            "const": "drugs",
-            "description": "The pharmacokinetics and the pharmacodynamics pathway elicited by the administration of specific drugs.",
-            "source": "http://bioportal.bioontology.org/ontologies/PW/?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPW_0000754"
         }
     ]
-}
+}
```

```suggestion
}

```
The file should have an empty line at the end. Some file types will not open correctly without it.

---

Noted, thank you!